src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -264,6 +264,7 @@ protected static function photoEmbed($arguments, $src)
      */
     private static function buildAttributeListFromArguments(array $arguments, array $exclude = []): ArrayList
     {
+        Deprecation::notice('4.5.0', 'Use {$Arguments.name} directly in shortcode templates to access argument values');
```

Comment:
Since when do shortcodes use templates? I think we need to find out what's going on here and update the message to be clearer.